task5/Task5_new/src/main/java/bsu/practice/Task5_new/PageServlet.java

```diff
@@ -0,0 +1,21 @@
+package bsu.practice.Task5_new;
+
+import javax.servlet.*;
+import javax.servlet.http.*;
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Scanner;
+
+public class PageServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        response.setContentType("text/html");
```

Comment:
Well, this is some solution. Please be only aware that such won't be tolerated in prod because contains significant overhead (IO operations are small, and no caching anywhere around). In prod you'd be expected to use forwarding or redirecting. But, anyway, formally, the task is done)